src/authentication/user.authentication.ts

```diff
@@ -0,0 +1,67 @@
+import { Request, Response, NextFunction } from "express";
+import passport from "passport";
+
+// Didn't work to have it in the global file.
+declare global {
+  namespace Express {
+    interface User {
+      username: string;
+      id?: number;
```

Comment:
I may have an alternative fix for this, we talk on Thursday about it.

But like, in what corner of the dark web did you find this?

Reply:
Haha I think this is one of the recommended way of doing it, pretty sure it was one of the most rated answers on stack overflow for the error msg, it seems logical to me to extend the interface.
Either way would love to see an alternative solution!